packages/dendron-viz/src/should-exclude-path.ts

```diff
@@ -0,0 +1,29 @@
+import { isMatch } from "micromatch";
```

Comment:
For this and other files, could we follow `camelCase` for the file name instead of `kebab-case`? I think that's the standard in the code base.